rules/azure/defense_evasion_event_hub_deletion.toml

```
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2020/08/18"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an Event Hub deletion in Azure. An Event Hub is an event processing service that ingests and processes large
+volumes of events and data. An adversary may delete an Event Hub in an attempt to evade defenses.
```

Minor suggestion. Maybe detection would be more accurate?

```suggestion
volumes of events and data. An adversary may delete an Event Hub in an attempt to evade detection.
```